setup.cfg

```diff
@@ -1,3 +1,65 @@
+[metadata]
+name = ovh
+description = "Official OVH.com API wrapper"
+long_description = file: README.rst
+version = 0.4.8
+author = Jean-Tiare Le Bigot
+author_email = jean-tiare.le-bigot@corp.ovh.com
+url = http://api.ovh.com
+license = BSD
+keywords = ovh, sdk, rest
+classifiers =
+    License :: OSI Approved :: BSD License
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: System :: Archiving :: Packaging
+
+[options]
+packages = find:
+setup_requires =
+    setuptools>=30.3.0
+include_package_data = True
+
+[options.entry_points]
+console_scripts = 
+    octodns-compare = octodns.cmds.compare:main
```

Comment:
Why add octodns as an extra package ?
Not necesssary in this library

Reply:
Just mistake